### DISCUSSION TITLE
export a selection of usd stage as fbx

### DISCUSSION THREAD

**Discussion by jhultgre** [Autodesk/maya-usd#1506] [Q&A]
My artists are asking if there is a way to selected some nodes inside the usd shape and export those as a fbx file? They want export part of it to hand off to another team.

**Comment by kxl-adsk** [comment]
MayaUSD allows you to import from an SDF layer in memory and you can leverage UFE global selection list to import only what artist selected. Once done, then you can do export these newly imported objects to FBX.

I don't have a minimal script to illustrate how to implement it - do you think you have enough to start or would need an example?

Cheers,
KxL

**Reply by jhultgre** [reply]
Is there UFE documentation online anywhere?

**Reply by ppt-adsk** [reply]
Hi @jhultgre, have a look here:
https://help.autodesk.com/view/MAYAUL/2022/ENU/?guid=Maya_SDK_ufe_ref_index_html
Are you on Maya 2022 or Maya 2020?  Maya 2020 has UFE v1 only, which is far more limited than the UFE v2 we have in 2022.

Hope this helps.

**Reply by jhultgre** [reply]
We're still on 2020 for a while

**Reply by ppt-adsk** [reply]
If all you need is to look at the UFE selection and inspect UFE scene items, you should be fine with 2020 / UFE v1.  Let us know if it isn't the case, we'll try to help out.  The mayaUsd unit tests are a useful source of inspiration for how to use UFE:
https://github.com/Autodesk/maya-usd/tree/dev/test/lib/ufe
You might find testSelection.py to be useful.  Cheers.
